pkg/mover/chart.go

```diff
@@ -31,6 +31,19 @@ var (
 	ErrOCIRewritesMissing = errors.New("at least one rewrite rule is required")
 )
 
+type ChartLoadingError struct {
+	Path  string
```

Comment:
do these properties need to be exposed? It seems that you implemented at the bottom the errors interface so there might be no reason to do so.